packages/@okta/vuepress-site/docs/concepts/event-hooks/index.md

```diff
@@ -103,7 +103,7 @@ As a best practice, you should return the HTTP response immediately, rather than
 
 ### Rate limits
 
-Event hooks aren't triggered after reaching a limit of 400,000 applicable events, per org, per day.
+There is a limit of 400,000 applicable events, per org, per 24-hour period after which event hooks won't trigger. The system log receives a warning prior to hitting the event limit when the number of events hits 280,000. The event limit resets 24 hours after the first event is sent.
```

Comment:
Slight rewording.
```suggestion
Okta limits each org to 400,000 applicable events within a 24-hour period. After reaching this threshold, further event hooks aren't triggered. An event is no longer counted against this limit after 24 hours. Okta records a warning to your system log when you reach 280,000 events. 
```

Reply:
I like the first re-wording, but would like to keep the latter two sentences as they address the issue raised by the customer, namely, when does the reset occur.